ReadMe.md

```diff
@@ -0,0 +1,20 @@
+# Settings
+
+## Contents 
+
+- [`src/`](./src)
+    - The main event - the code for this project!
+- [`scripts/`](./scripts/ReadMe.md)
+    - Helpful scripts to build/test/run the project!
+
+## Getting started 
+
+1. Clone the project 
+1. Install project requirements 
+    - [`.NET Core SDK 1.0.4`](https://github.com/dotnet/core/blob/master/release-notes/download-archives/1.1.2-download.md)
+    - [`Node LTS`](https://nodejs.org/en/download/)
+1. Build and run the project
+    - `scripts/ci`
```

Comment:
looks like you renamed this to scripts/cibuild

Reply:
Ah, good catch - fixed this via https://github.com/jopache/Settings/pull/18/commits/18ddc3bf716e98130f580caaeeffb3903298da50